src/main/java/org/jboss/msc/service/ServiceControllerImpl.java

```diff
@@ -325,12 +325,12 @@ private boolean shouldStop() {
      * Returns true if controller is in rest state and no async tasks are running, false otherwise.
      * @return true if stable rest state, false otherwise
      */
-    boolean isStableRestState() {
+    private boolean isStableRestState() {
```

Comment:
Package-private is adequate encapsulation.  When you have a private method on Java 9 and earlier which is accessed from a nested class, the compiler has to produce an "access" method which is package-private anyway, so any improved encapsulation is an illusion.